bin/ch-test

```diff
@@ -383,6 +411,16 @@ if [[ ! -d $CHTEST_EXAMPLES_DIR ]]; then
     fatal "examples not found: $CHTEST_EXAMPLES_DIR"
 fi
 
+# Is the user a contributor? (Obtaining an e-mail address this way is flaky,
+# but it works at Los Alamos, where the team is based.)
+email="$USER@$(hostname --domain)"
```

Comment:
We should consider using `git config --get user.email` to get the users email address. I would never have pedantic mode enforced, because I usually develop in a VM that doesn't have my expected username or domain. I am likely not the only one in this case. I would expect despite past blunders that every one of us has our git configured to our contributor email in the global config.

Reply:
![parks & rec mic drop](https://media.giphy.com/media/UmBdALbYTmCJ2/source.gif)